src/docs/development/ui/layout/responsive.md

```diff
@@ -48,19 +42,24 @@ apps with responsive design:
 
 Other useful widgets and classes for creating a responsive UI:
 
-* [`AspectRatio`][]
-* [`CustomSingleChildLayout`][]
-* [`CustomMultiChildLayout`][]
-* [`FittedBox`][]
-* [`LayoutBuilder`][]
-* [`MediaQuery`][]
-* [`MediaQueryData`][]
-* [`OrientationBuilder`][]
+* [`AspectRatio`][] API and
```

Comment:
How about something like this, to make the list more natural:

* [`AspectRatio`](...) (also see: [Widget of the Week video](...))
* [`CustomSingleChildLayout`](...)

Reply:
Oops, the link text for the videos shouldn't all be the same, for accessibility reasons. But you get the gist.